README.rst

```diff
@@ -270,6 +270,21 @@ Config settings
                 root /www/resource_cache;
             }
 
+6.  If you are not serving your archived resources through a webserver, you can upload them to ``s3filestore``. 
+    Make sure you have properly setup `ckanext-s3filestore <https://github.com/datopian/ckanext-s3filestore>`_ for this.
+
+    Add the settings to the CKAN config file:
+
+      * ``ckanext.archiver.s3upload_enable`` = ``True`` to enable upload to filestore. If unset or ``False``, the resources are going to be archived locally instead of on s3filestore.
```

Comment:
``ckanext.archiver.s3upload_enable`` = ``True`` to enable upload to cloud storage, defaluts to `False`